casper/src/main/scala/coop/rchain/casper/util/comm/CommUtil.scala

```diff
@@ -47,9 +44,9 @@ object CommUtil {
     new CommUtil[F] {
 
       def sendBlockHash(hash: BlockHash, blockCreator: ByteString): F[Unit] =
-        sendToPeers(BlockHashMessageProto(hash, blockCreator)) <* Log[F].info(
+        Log[F].info(
           s"Sent hash ${PrettyPrinter.buildString(hash)} to peers"
-        )
+        ) >> sendToPeers(BlockHashMessageProto(hash, blockCreator))
```

Comment:
Is there a reason why logs are now printed before the executing functions? It makes sense to know when the call finished. It's similar with other two changes in this file.

Reply:
https://github.com/rchain/rchain/pull/2902/commits/b94764d552d88a30e0bea77be8c951688d9024ca